runtime/tests/checker/builtinfunctions_test.go

```diff
@@ -85,6 +85,23 @@ func TestCheckToBytes(t *testing.T) {
 	})
 }
 
+func TestCheckAddressFromBytes(t *testing.T) {
```

Comment:
Added test cases.

I do have one question regarding labelled arguments. I have declared the function with `ArgumentLabelNotRequired`. My assumption was that the label would be optional but user would still be able to provide that if they want.

Based on the tests, I've observed this to not be the case. Even if the user provides the correct label (`bytes` for this case), we seem to be throwing an error - https://github.com/onflow/cadence/blob/2abe1cb05f6ac90f148ab90b4f492486c318f5ea/runtime/sema/check_invocation_expression.go#L326-L341. Am I missing something or is that by design?

Reply:
The argument label is not optional, either the label is mandatory (if the label is specified), or shouldn't have at all (if `ArgumentLabelNotRequired`)

Reply:
> The argument label is not optional, either the label is mandatory (if the label is specified), or shouldn't have at all (if ArgumentLabelNotRequired)

@SupunS  is this introduced recently ?

Reply:
When a parameter is declared with `_`, that declares that a call _must not_ provide a label for the parameter. For example, a function declaration `fun double(_ n: Int): Int` must be called like `double(1)` and cannot be called like `double(n: 1)`.

When a parameter is declared with an explicit argument label, that declares that a call _must_ provide the label. For example, a function declaration `fun double(number n: Int): Int` must be called like `double(number: 1)` and cannot be called like `double(1)` or `double(n: 1)`.

When a parameter is declared without an argument label, that declares that the parameter name is also the argument label and that a call _must_ provide the parameter name as the label. For example, a function declaration `fun double(number: Int): Int` must be called like `double(number: 1)` and cannot be called like `double(1)`.

This feature has been there since the inception of Cadence, and is directly taken from Swift: https://docs.swift.org/swift-book/documentation/the-swift-programming-language/functions/#Function-Argument-Labels-and-Parameter-Names

Unfortunately, this behaviour was not correctly implemented for calls of nested members. We fixed this a while ago in https://github.com/onflow/cadence/pull/1717. Because the fix is effectively a breaking change, we will only release it as part of the Stable Cadence milestone. See "Missing or Incorrect Argument Labels Not Reported" in https://forum.onflow.org/t/another-update-on-stable-cadence/3715.

Reply:
thanks a lot @turbolent